checks/51-rpmlint

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+RPMLINT_CHROOT="$BUILD_ROOT/usr/lib/rpmlint-image"
+WORKDIR=$RPMLINT_CHROOT/RPM
```

Comment:
AFAICT, this is never cleaned up? Maybe you can use bind mounts instead of copying, which should also be faster.

Reply:
Do you mean mounting `/home/abuild/rpmbuild/` or individual locations:
```
/home/abuild/rpmbuild/SOURCES
/home/abuild/rpmbuild/RPMS
/home/abuild/rpmbuild/SRPMS
```
?

Reply:
The less mounts, the better.